### ISSUE TITLE
Lip Cyanosis

### ISSUE THREAD

**Issue by cmrn-rhi** [obophenotype/human-phenotype-ontology#7125]
**Preferred term label:**
lip cyanosis

**Synonyms**
cyanotic lips

_layperson term:_
bluish lips

**Definition (free text, please give PubMed ID)**
Persistent blue discoloration of the lips.

**Parent term (use hpo.jax.org/app)**
Facial Cyanosis (new term request #7124)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
COVID-19

**Your nano-attribution (ORCID)**
0000-0002-9578-0788

**Comment by pnrobinson** [comment]
added, thanks
